src/lib/components/data-table/DataTable.svelte

```
@@ -33,29 +42,45 @@
 		renderExpandedRow,
 		emptyStateMessage = 'No results.',
 		emptyStateContent,
-		expandColumnId
+		expandColumnId,
+		enableSelection = false,
+		onSelectionChange,
+		getRowId = (row: any) => String(row.id),
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)